packages/sdk/src/sampler.ts

```diff
@@ -83,14 +83,24 @@ export class SwSampler implements Sampler {
     if (decisions.status > oboe.TRACING_DECISIONS_OK) {
       this.logger.warn(
         "oboe decisions returned with an error status",
-        new OboeError("Context", "getDecisions", decisions.status),
-      )
-      this.logger.debug(
-        decisions.status_msg,
-        decisions.auth_msg,
-        decisions.status,
+        new OboeError(
+          "Context",
+          "getDecisions",
+          decisions.status,
+          decisions.status_msg,
+        ),
       )
       return { decision: SamplingDecision.NOT_RECORD }
+    } else if (decisions.auth > oboe.TRACING_DECISIONS_AUTH_OK) {
+      this.logger.warn(
+        "oboe decisions auth returned with an error status",
```

Comment:
This could get noisy if the application is receiving requests with a misconfigured (or just "cross account") DEM probe that keeps sending trigger trace with an invalid signature. In this case the impact is just that we don't sample the request, so in my mind can be a lower log level.